.github/workflows/test.yml

```
@@ -34,4 +34,6 @@ jobs:
         pytest life_expectancy --cov
     - name: Lint with pylint
       run: |
-        pylint life_expectancy.cleaning
+        pylint life_expectancy.main
+        pylint life_expectancy.data_loading
+        pylint life_expectancy.data_cleaning
```

```suggestion
        pylint life_expectancy
```

Incluira tamb]em os testes.